harness/ts/harness.ts

```diff
@@ -106,7 +107,10 @@ export class Runner<W extends Workflow, A extends ActivityInterface> {
       taskQueue: options.taskQueue,
     };
     if (options.nodeModulesPath) {
-      workerOpts.nodeModulesPaths = [options.nodeModulesPath];
+      const ourNodeMods = path.join(process.cwd(), 'node_modules');
+      workerOpts.nodeModulesPaths = [options.nodeModulesPath, ourNodeMods];
+      // TODO: Remove this `as any` once published version has this addition
+      (workerOpts as any).typescriptContextPath = process.cwd();
```

Comment:
Another option would be to compile typescript ahead of time and bundle the generated JS.